distributions/debian/jamulus.install

```diff
@@ -1,3 +1,4 @@
 usr/bin/jamulus
 usr/share/applications/jamulus.desktop
+usr/share/applications/jamulus-server.desktop
```

Comment:
Unfortunately it doesn't show the server icon for me. Probably the icon is not set correctly?

Reply:
There's no .png in the tree, I'll check whether the .svgs can be used.

Reply:
Ah yes. .svg's would be even better.

Reply:
The .svgs were already installed by the `Makefile`, but to the wrong location (IMO). I've fixed that as well and adjusted the debian `.install` file. Can you test again once the build is done?